docs/deploy-configure.md

```diff
@@ -16,6 +16,11 @@ The [`/config/samples`](../config/samples) directory contains example MongoDBCom
 
 ## Deploy a Replica Set
 
+**Warning:** When you delete MongoDB resources, persistent volumes remain.
```

Comment:
```suggestion
**Warning:** When you delete MongoDB resources, persistent volumes remain. This is to ensure no unintended data-loss occurs.
```
If possible I would emphasise the reason for this, in case this would raise questions about why they are not deleted.